wire/msgcfilter.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/decred/dcrd/chaincfg/chainhash"
+)
+
+const (
+	// MaxCFilterDataSize is the maximum byte size of a committed filter.
+	// The maximum size is currently defined as 256KiB.
+	MaxCFilterDataSize = 256 * 1024
+)
+
+// MsgCFilter implements the Message interface and represents a cfilter message.
+// It is used to deliver a committed filter in response to a getcfilter
+// (MsgGetCFilter) message.
+type MsgCFilter struct {
+	BlockHash  chainhash.Hash
+	FilterType FilterType
+	Data       []byte
+}
+
+// BtcDecode decodes r using the wire protocol encoding into the receiver.
+// This is part of the Message interface implementation.
+func (msg *MsgCFilter) BtcDecode(r io.Reader, pver uint32) error {
+	// Read the hash of the filter's block
+	err := readElement(r, &msg.BlockHash)
+	if err != nil {
+		return err
+	}
+	// Read filter type
```

Comment:
All the other decode funcs have blank lines between each field, but this one doesn't.